src/web/comment/store/commentStore.ts

```diff
@@ -161,6 +161,11 @@ export const deleteComment = (commentId: string) => {
   );
 };
 
+export const resetComment = () => {
```

Comment:
```suggestion
export const resetComments = () => {
```

I think plural "comments" makes more sense for the name, because we're not just resetting one comment

Reply:
I agree, using the plural form makes more sense.